js/noms/src/assert-type.js

```diff
@@ -111,10 +115,17 @@ export function isSubtypeInternal(requiredType: Type<any>, concreteType: Type<an
   invariant(false);
 }
 
+// compoundSubtype is called when comparing the element types of two compound types. This is the
```

Comment:
For now the API doc picks up all `/** */`. I'm working on a system that follows the exported bindings (+methods) and then we should probably use JSDoc comments here.